src/style/_information.scss

```diff
@@ -32,53 +29,105 @@
             color: $info-link-font-color--hover;
         }
     }
-    > div
-    {
-        margin: 0 0 .5em 0;
 
-        &.description {
-            margin: 0 0 2em 0;
-        }
+    .url {
```

Comment:
**Change request**: I think this would be better as a BEM element on the info block, e.g. `.info__url` (unless there's a reason for `.url`?)